SystemInfo/Collector/JsonComposerLockSystemInfoCollector.php

```diff
@@ -54,8 +54,10 @@ public function build()
 
         $packages = [];
         $lockData = json_decode(file_get_contents($this->lockFile), true);
+        $minimumStability = isset($lockData['minimum-stability']) ? $lockData['minimum-stability'] : null;
+
         foreach ($lockData['packages'] as $packageData) {
-            $stability = null;
+            $stability = $minimumStability;
```

Comment:
I actually disagree... I don't see why the UI would need to do anything about it. 

The composer doc says that `minimum-stability` is stable unless specified otherwise. It seems to me that it is the responsibility of this collector to handle this.